smarts/core/sensors.py

```diff
@@ -291,6 +292,11 @@ def observe(sim, agent_id, sensor_state, vehicle):
             sim, agent_id, vehicle, sensor_state
         )
 
+        print(f'observe steps: {sensor_state.steps_completed}')
+        print(f'observe done: {done}')
+        if done and sensor_state.steps_completed == 1:
+            logger.warning('Agent is done at the first step')
```

Comment:
The print statements here show this for test_example:
`observe steps: 0
observe done: True
observe steps: 0
observe done: True
observe steps: 0
observe done: True
observe steps: 0
observe done: True
observe steps: 1
observe done: False
observe steps: 1
observe done: False
observe steps: 1
observe done: False
observe steps: 1
observe done: False
observe steps: 2
observe done: False
observe steps: 2
observe done: False
observe steps: 2
observe done: False
observe steps: 1
observe done: True
WARNING:smarts.core.sensors:Agent is done at the first step
observe steps: 2
observe done: False
observe steps: 3
observe done: False
observe steps: 3
observe done: False
observe steps: 3
observe done: False
observe steps: 3`

It shows this for using a normal example like single agent:
`observe steps: 0
observe done: True
observe steps: 0
observe done: True
observe steps: 0
observe done: True
observe steps: 0
observe done: True
observe steps: 1
observe done: False
observe steps: 1
observe done: False
observe steps: 1
observe done: False
observe steps: 1
observe done: False
observe steps: 2
observe done: False
observe steps: 2
observe done: False
observe steps: 2
observe done: False
observe steps: 2
observe done: False
observe steps: 1
observe done: False
observe steps: 3
observe done: False
observe steps: 3
observe done: False
observe steps: 3
observe done: False
observe steps: 3
observe done: False
observe steps: 2
observe done: False
observe steps: 4
observe done: False
observe steps: 4
observe done: False
observe steps: 4
observe done: False
observe steps: 4
observe done: False
observe steps: 3
observe done: False
observe steps: 5
observe done: False
observe steps: 5
observe done: False
observe steps: 5
observe done: False
observe steps: 5
observe done: False
observe steps: 4
observe done: True`